### PR TITLE
Jack's Greeting

### PR DESCRIPTION
**Pull request by jmahoney5**
I added a greeting for Jack.

### CODE REVIEW

**Review by KenBroden — DISMISSED** [review]
Looks good

**Review by mahon479 — DISMISSED** [review]
Everything looks correct - good job.

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -9,12 +9,18 @@ public class Hellos {
...
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
 
     return builder.toString();
   }
 
+  private String jackSaysHello() {
+    // TODO Auto-generated method stub
+    return "Jack says 'Hello!'\n";
+  }
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: You should remove comments like the `TODO` comments that get generated by the system.

```suggestion
  private String jackSaysHello() {
    return "Jack says 'Hello!'\n";
  }
```

**Review by NicMcPhee — CHANGES_REQUESTED** [review]

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -23,7 +24,12 @@ public String generateOutput() {
     return builder.toString();
   }
 
-  private String malenaSaysHello() {
+
+  private String jackSaysHello() {
+    return "Jack says 'Hello!'\n";
+  }
+  
+    private String malenaSaysHello() {
```
Comment (KenBroden) [inline src/main/java/hellos/Hellos.java]: Really small thing, but it looks like you have indented the malenaSaysHello method after putting in your method

Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java]: This is important observation – thanks to @KenBroden for noting this.

**Review by NicMcPhee — DISMISSED** [review]

**Review by KenBroden — DISMISSED** [review]

**Review by NicMcPhee — APPROVED** [review]

**Review by ethangraybar — APPROVED** [review]
Looks good!

**Pull request merged**